packages/docker-utils/vitest.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'vitest/config';
```

Comment:
Can you explain why you went with this approach rather than a shared top-level vitest config?

Reply:
I spent some time trying to make Vitest's projects/workspaces work with a single `vitest.config.ts` file in the root of the repository. This is as close as I got:

```ts
import { defineConfig } from 'vitest/config';

export default defineConfig({
  test: {
    root: import.meta.dirname,
    workspace: ['packages/*'],
    coverage: {
      reporter: ['html', 'text-summary', 'cobertura'],
      all: true,
      include: ['src/**'],
    },
  },
});
```

Then, one could update the `test` script in a given package to something like `vitest run --coverage --project @prairielearn/csv`. This gets us close, however, coverage is broken in that case.

@reteps and I discussed this offline and agreed to merge this PR as-is as an iterative step forward.

Reply:
Further notes for why the config is duplicated instead of using `mergeConfig`:

> We hadn't touched the previous coverage config since it was introduced. IMO the only value to DRY in this case is making it easy to change everything, but I don't anticipate it changing.

...

> Not a fan of a separate package either. Let’s just roll with the duplication for now.